src/pmpo_c.cpp

```diff
@@ -594,7 +588,7 @@ void polympo_getMeshVtxRotLat_f(MPMesh_ptr p_mpmesh, const int nVertices, double
   }
 }
 
-void polympo_setMeshVel_f(MPMesh_ptr p_mpmesh, const int nComps, const int nVertices, const double* array){
+void polympo_setMeshVel_f(MPMesh_ptr p_mpmesh, const int nComps, const int nVertices, const double* uVelIn, const double* vVelIn){
```

Comment:
we do not need 'nComps'

Reply:
removed https://github.com/SCOREC/polyMPO/commit/8bec689310aa9db8a2ecd8673a002858a0506d28